examples/tensorflow/classification/main.py

```diff
@@ -299,13 +304,32 @@ def run(config):
         write_metrics(results[1], config.metrics_dump)
 
     if 'export' in config.mode:
-        save_path, save_format = get_saving_parameters(config)
-        compression_ctrl.export_model(save_path, save_format)
-        logger.info('Saved to {}'.format(save_path))
+        export_model(compression_ctrl, config)
 
     close_strategy_threadpool(strategy)
 
 
+def export_model(compression_ctrl, config):
+    save_path, save_format = get_saving_parameters(config)
+    if config.prepare_for_inference:
+        model = compression_ctrl.prepare_for_inference()
```

Comment:
It looks like you called `prepare_for_inference()` twice for the model in case `train` + `export`. Could you fix it?